### PR TITLE
make the button a measly button

### PR DESCRIPTION
**Pull request by jamesgorrie**
As noticed by @theefer

### CODE REVIEW

File: kahuna/public/js/components/gu-date-range/gu-date-range.html [hunk 1]
```diff
@@ -21,6 +21,7 @@ <h2 class="gu-date-range__overlay__title search__overlay__title">Presets</h2>
         <div class="gu-date-range__overlay__presets">
             <button type="button"
                     class="gu-date-range__overlay__preset__button button"
+                    type="button"
```
Comment (theefer) [inline kahuna/public/js/components/gu-date-range/gu-date-range.html]: There is already a `type="button"` above.

**Comment by paperboyo** [comment]
A platonic :+1:

File: kahuna/public/js/components/gu-date-range/gu-date-range.html (hunk 1 above)
Comment (theefer) [inline kahuna/public/js/components/gu-date-range/gu-date-range.html]: I think you meant to add this to the first `button` in this file instead.

Comment (jamesgorrie) [inline kahuna/public/js/components/gu-date-range/gu-date-range.html]: Dammit, thanks :+1: to peer review and git history rewrite.

**Comment by jamesgorrie** [comment]
@paperboyo almost right.

**Comment by theefer** [comment]
:+1:

**Comment by theefer** [comment]
:eyes:

**Pull request merged**